tests/local/network.go

```diff
@@ -128,9 +126,9 @@ func NewLocalNetwork(
 	avalancheGoBuildPath, ok := os.LookupEnv("AVALANCHEGO_BUILD_PATH")
 	Expect(ok).Should(Equal(true))
 
-	ctxWithTimeout, cancelBootstrap := context.WithTimeout(ctx, timeout)
+	ctx, cancelBootstrap := context.WithCancel(ctx)
```

Comment:
Not related to your change here (and I probably wrote this code myself 😅 ) but shouldn't the `defer` come _before_ the context is passed into `BootstrapNewNetwork`?

Reply:
Indeed -- thanks for noticing